source/upgrade.txt

```diff
@@ -151,6 +151,15 @@ The v4.0 driver introduces the following breaking changes to the API:
   library equivalents. For context, see the :ref:`Build System Breaking Changes
   <version-4.0-build-system-breaking>` section.
 
+- Removes the following ``using`` directives and declarations:
```

Comment:
```suggestion
- Removes the following undocumented ``using`` directives and declarations:
```

Suggest including this word to indicate users were not expected to be using these (undocumented) interfaces.